File: king-admin-java/src/main/java/com/oukingtim/service/SysRoleService.java
package com.oukingtim.service;

import com.baomidou.mybatisplus.service.IService;
import com.oukingtim.domain.SysRole;

import java.util.List;

/**
 * Created by oukingtim
 */
public interface SysRoleService extends IService<SysRole> {
    /**
     * 根据用户id查询角色集合
     * @param userId
     * @return
     */
    List<SysRole> getList(String userId);
}

File: king-admin-kotlin/src/main/kotlin/com/oukingtim/service/SysRoleService.kt
package com.oukingtim.service

import com.baomidou.mybatisplus.service.IService
import com.oukingtim.domain.SysRole

/**
 * Created by oukingtim
 */
interface SysRoleService : IService<SysRole> {
    /**
     * 根据用户id查询角色集合
     * @param userId
     * *
     * @return
     */
    fun getList(userId: String): List<SysRole>
}